tests/test_models/test_models.py

```diff
@@ -374,3 +376,44 @@ def test_thaw_bad_param(self, mock_model):
         fr = mock_model.frozen
         mock_model.thaw("pinguino")
         assert all([old == new for old, new in zip(fr, mock_model.frozen)])
+
+    def test_normalize(self, mock_model):
+        np.random.seed(123)
+        F1, cov1 = mock_model()
+        mock_model.norm = True
+        np.random.seed(123)
+        F2, cov2 = mock_model()
+        factor = mock_model.emulator.norm_factor(mock_model.grid_params)
+        assert np.allclose(F1 * factor, F2)
```

Comment:
@iancze @gully how should the covariances relate here?
`cov1 \approx cov2 * factor **2` **seems** appropriate but is not true upon testing. Is my intuition wrong or is there a bug I need to resolve?

Reply:
Naively, I would expect this to be what you said. That's not to say my intuition couldn't be wrong here either, and maybe it's worth coding up a [simple example](https://en.wikipedia.org/wiki/Covariance). `factor` is a scalar here, right?